contracts/DandelionOrg.sol

```diff
@@ -406,81 +406,81 @@ contract DandelionOrg is BaseTemplate {
+        DeployedContracts storage contracts = deployedContracts[msg.sender];
+        require(contracts.token != address(0), ERROR_MISSING_TOKEN_CONTRACT);
 
-        MiniMeToken token = MiniMeToken(c.token);
+        MiniMeToken token = MiniMeToken(contracts.token);
         return token;
     }
 
-    function _popBaseAppsCache() internal returns (
+    function _getBaseApps() internal returns (
         Finance finance,
         TokenManager tokenManager,
         Vault vault
     )
     {
-        Cache storage c = cache[msg.sender];
-        require(c.dao != address(0), ERROR_MISSING_CACHE);
+        DeployedContracts storage contracts = deployedContracts[msg.sender];
+        require(contracts.dao != address(0), ERROR_MISSING_CONTRACT);
 
-        finance = Finance(c.finance);
-        tokenManager = TokenManager(c.tokenManager);
-        vault = Vault(c.agentOrVault);
+        finance = Finance(contracts.finance);
+        tokenManager = TokenManager(contracts.tokenManager);
+        vault = Vault(contracts.agentOrVault);
     }
 
-    function _popAgentAsVaultCache() internal returns (bool agentAsVault) {
-        Cache storage c = cache[msg.sender];
-        require(c.dao != address(0), ERROR_MISSING_CACHE);
+    function _getAgentAsVault() internal returns (bool agentAsVault) {
+        DeployedContracts storage contracts = deployedContracts[msg.sender];
+        require(contracts.dao != address(0), ERROR_MISSING_CONTRACT);
 
-        agentAsVault = c.agentAsVault;
+        agentAsVault = contracts.agentAsVault;
     }
 
-    function _clearCache() internal {
-        Cache storage c = cache[msg.sender];
-        require(c.dao != address(0), ERROR_MISSING_CACHE);
+    function _clearDeployedContracts() internal {
+        DeployedContracts storage contracts = deployedContracts[msg.sender];
+        require(contracts.dao != address(0), ERROR_MISSING_CONTRACT);
 
-        delete c.dao;
-        delete c.token;
-        delete c.finance;
-        delete c.tokenManager;
-        delete c.agentOrVault;
-        delete c.agentAsVault;
+        delete contracts.dao;
+        delete contracts.token;
+        delete contracts.finance;
+        delete contracts.tokenManager;
+        delete contracts.agentOrVault;
+        delete contracts.agentAsVault;
     }
 
-    function _ensureBaseAppsCache() internal {
-        Cache storage c = cache[msg.sender];
-        require(c.finance != address(0), ERROR_MISSING_CACHE);
-        require(c.tokenManager != address(0), ERROR_MISSING_CACHE);
-        require(c.agentOrVault != address(0), ERROR_MISSING_CACHE);
+    function _ensureBaseAppsDeployed() internal {
+        DeployedContracts storage contracts = deployedContracts[msg.sender];
+        require(contracts.finance != address(0), ERROR_MISSING_CONTRACT);
+        require(contracts.tokenManager != address(0), ERROR_MISSING_CONTRACT);
+        require(contracts.agentOrVault != address(0), ERROR_MISSING_CONTRACT);
```

Comment:
I think these would be better as different errors, one for each. Up to you though.